__tests__/plugins_phones.test.js

```diff
@@ -5,27 +5,27 @@ describe("phones_plugin", () => {
         it("should call grabHrefTel", () => {
             const lines = ["tel:'+44 1632 960983'"];
             const plugin = new PhonesPlugin();
-            plugin.grabHrefTel = jest.fn();
+            plugin.grabHrefTel = jest.fn(() => []);
 
             plugin.main(lines);
-            expect(plugin.grabHrefTel.mock.calls.length).toEqual(lines.length);
+            expect(plugin.grabHrefTel.mock.calls.length).toEqual(1);
         });
         it("should return results", () => {
             const phones = [
-                "href='tel:+44 1632 960983'",
-                "+44 1632 960984"
+                `href="tel:+44 161 414 1080" <tag>`,
+                `+44 161 414 1080`
```

Comment:
Use of ` and ' and " seems really inconsistent

Reply:
IMO, should be backticks for strings with vars in

double quotes anywhere else